CODEOWNERS

```diff
@@ -39,19 +39,8 @@
 go.sum @fleetdm/go
 go.mod @fleetdm/go
 /cmd/ @fleetdm/go
-/orbit/ @lucasmrod @getvictor @roperzh @gillespi314
 /server/ @fleetdm/go
-/server/service/handler.go @lucasmrod @getvictor @roperzh @gillespi314
-/server/mdm/ @roperzh @gillespi314 @lucasmrod @georgekarrv
-/server/worker/ @lucasmrod @getvictor @roperzh @gillespi314 @lukeheath
-/server/vulnerabilities/ @lucasmrod @mostlikelee @getvictor @lukeheath
-/server/cron/ @getvictor @lucasmrod @roperzh @mostlikelee @lukeheath
-/ee/fleetd-chrome @lucasmrod @getvictor @RachelElysia
```

Comment:
Folders without code owners will be open for all fleeties.
That's the intension. right?

Reply:
For example, the vulnerabilities directory that impacts the artifact build process.

Reply:
@lukeheath SGTM. Let me do a quick look to see if I have any such folders

Reply:
maybe the only missing is `/ee/server` for @\fleetdm/go

Reply:
@roperzh Good catch, added.